### DISCUSSION TITLE
Clockwise vs. Anticlockwise drawing

### DISCUSSION THREAD

**Discussion by holgerlembke** [dune3d/dune3d#91] [Q&A]
I follow the beginner intro and draw this:

![grafik](https://github.com/dune3d/dune3d/assets/2034993/42cae286-063b-4320-81a8-1211e8bfcf33)

Beginning bottom left, up, arc, down, close.

I get this if I begin bottom right, up, arc, down, close.

![grafik](https://github.com/dune3d/dune3d/assets/2034993/4558f782-e2ee-4c7f-b222-b6447a9a5633)

Notice the difference, as far as I understand, there is one degree of freedom more/less.

(I am totally unsure why and what that mean. At least it is totally confusing /me to follow the intro.)

**Comment by holgerlembke** [comment]
Addendum: it is confusing. I get "random" results. Sometimes 6, sometimes 5, sometimes 4. 

So it looks like I miss/overlook some very basic function here.

**Comment by carrotIndustries** [comment]
This should be addressed by https://github.com/dune3d/dune3d/commit/e9bbf5075513ffc9b7502ac90a8bbb780255fd2d

**Comment by holgerlembke** [comment]
Nice! I'll try it!
